src/bindings/python/src/pyopenvino/graph/ops/constant.cpp

```diff
@@ -59,6 +63,7 @@ std::vector<size_t> _get_strides(const ov::op::v0::Constant& self) {
     }
 }
 
+// TODO: Remove in future and re-use `get_data`
```

Comment:
Is this something we could add as a GFI?

Reply:
Might be, but this is distant future as these implementations are still used in opset1-12 for Constant op.